vocab_files/observable_properties_by_module/basal-area/basal-wedge-protocol/collection.ttl

```diff
@@ -0,0 +1,16 @@
+PREFIX dcterms: <http://purl.org/dc/terms/>
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX schema: <https://schema.org/>
+PREFIX skos: <http://www.w3.org/2004/02/skos/core#>
+
+<https://linked.data.gov.au/def/test/dawe-cv/84b0a152-e6f6-4ea2-b973-04238034bb52>
+    a skos:Collection ;
+    dcterms:description "A collection of properties in Basal Area module - Basal Wedge protocol" ;
+    rdfs:isDefinedBy <https://linked.data.gov.au/def/nrm> ;
+    skos:member
+        <https://linked.data.gov.au/def/test/dawe-cv/29b37ffc-9a41-44f7-889a-bab63b48fa93> ,
+        <https://linked.data.gov.au/def/test/dawe-cv/43178892-92a6-434f-9895-340700e299e6> ;
```

Comment:
Should `https://linked.data.gov.au/def/test/dawe-cv/3e0a5d97-c623-477e-98fe-8fe120907530` also be a member here?